ahmedkamals/foo-protocol-proxy
# :whale: Using docker template.

## What does this do?
Using docker file templates

## Why is it a good idea?
To ease the deployment.

## How I did it?
Added template files inside the **build** folder, and it should be replaced in the root directory after the deployment script is run.

## How to verify it?
- Run the deployment script.
- Check the root directory, there should be the generated files.

## Remarks


## Release note
NONE
```release-note

```
